src/triagebot/review-queue-tracking.md

```diff
@@ -0,0 +1,61 @@
+# Review queue tracking
+
+Triagebot supports more advanced tracking of reviewers' workload in the `rust-lang` repository. It tracks how many "relevant" pull requests are assigned to each reviewer, and allows reviewers to configure maximum capacity of such PRs, and also if they want to be automatically assigned or not.
+
+This pages describes how the review queue works and how you can interact with `triagebot` on Zulip to configure and examine the review queue.
+
+## Configuration
+
+To enable review queue tracking for a repository, include `[pr-tracking]` table in its `triagebot.toml`.
+
+To take the review queue into account when assigning reviewers on PRs, add a `[assign.review_prefs]` table to `triagebot.toml`.
+
+> Note that this functionality currently only works only for the `rust-lang/rust` repository (it is hardcoded in `triagebot`). Enabling it for more repositories requires additional design and implementation work.
+
+## Review queue design
+
+The review queue remembers how many "relevant" `rust-lang/rust` PRs are assigned to each reviewer at any given point in time.
+Currently, the heuristic for what makes a PR "relevant" works as follows:
+- The PR must not be blocked (it must not have the `S-blocked` or `S-inactive` labels).
+- The PR must not be a rollup.
+- The PR must be waiting for a reviewer (must have the `S-waiting-on-review` label).
+- The PR must be assigned to someone else than the PR author.
+- The PR must be open and not a draft.
+
+If a PR passes all these checks and it is assigned to reviewer `R`, it will be considered to be in `R`'s review queue.
+
+See the implementation of the [`waits_for_a_review`](https://github.com/rust-lang/triagebot/blob/HEAD/src/handlers/pr_tracking.rs#L289) function in triagebot for more details.
```

Comment:
This line number will likely point to the wrong place in the near future. I might suggest just linking to a specific hash, although that's not great, either.

```suggestion
See the implementation of the [`waits_for_a_review`](https://github.com/rust-lang/triagebot/blob/7044e5449aaf29eea939bc7db81c63b7c872b9b8/src/handlers/pr_tracking.rs#L277-L304) function in triagebot for more details.
```